### ISSUE TITLE
Where is the M-estimators?

### ISSUE THREAD

**Issue by lymhust** [ankurhanda/gvnn#15]
Hi,

Thanks for sharing this great work. 

I am trying to apply different loss functions. However, I have not found anything related to M-estimator? Can you help providing some information about M-estimators?

Thanks!

**Comment by ankurhanda** [comment]
could you use Huber here: https://github.com/viorik/extracunn/blob/master/Huber.cu

I'm writing a tf versoin of this so will release soon. torch is too old.

**Comment by vbalnt** [comment]
@ankurhanda 

Hi is there any update on the tensorflow version?

**Comment by ankurhanda** [comment]
Sorry busy with lots of other stuff at the moment! Will let you know when I do.

**Comment by ankurhanda** [comment]
@vbalnt have you seen this https://stackoverflow.com/questions/39106732/how-do-i-combine-tf-absolute-and-tf-square-to-create-the-huber-loss-function-in
